exports/exports_test.go

```diff
@@ -214,3 +307,22 @@ func setupTest(requestAppResources exports.RequestApplicationResources) chi.Rout
 
 	return router
 }
+
+func populateTestData() chi.Router {
+	// define router
+	router := setupTest(mockReqeustApplicationResouces)
+
+	for i := 1; i <= 3; i++ {
+		req := createExportRequest(
+			fmt.Sprintf("Test Export Request %d", i),
+			"json",
+			`{"application":"exampleApp", "resource":"exampleResource"}`,
+		)
+
+		rr := httptest.NewRecorder()
+
+		router.ServeHTTP(rr, req)
+	}
+
+	return router
+}
```

Comment:
Maybe we could try something like this, to make sure the filtering is working properly?

```
func populateTestData() chi.Router {
	// define router
	router := setupTest(mockReqeustApplicationResouces)

	for i := 1; i <= 3; i++ {
		req := createExportRequest(
			fmt.Sprintf("Test Export Request %d", i),
			"json",
			`{"application":"exampleApp", "resource":"exampleResource"}`,
		)

		rr := httptest.NewRecorder()

		router.ServeHTTP(rr, req)
	}

	oneDayAgo := time.Now().AddDate(0, 0, -1)
	oneDayFromNow := time.Now().AddDate(0, 0, 1)

	modifyExportCreated("Test Export Request 1", oneDayAgo)
	modifyExportCreated("Test Export Request 2", oneDayFromNow)
	modifyExportExpiry("Test Export Request 3", oneDayFromNow)

	return router
}

func modifyExportCreated(exportName string, newDate time.Time) {
	testGormDB.Exec("UPDATE export_payloads SET created_at = ? WHERE name = ?", newDate, exportName)
}

func modifyExportExpiry(exportName string, newDate time.Time) {
	testGormDB.Exec("UPDATE export_payloads SET expires_at = ? WHERE name = ?", newDate, exportName)
}
```

Reply:
Added it!